lighthouse-core/test/lib/tap-target-rect-helpers-test.js

```diff
@@ -0,0 +1,246 @@
+/**
```

Comment:
nit: can we get this filename aligned with the source one? I like your `tappable-rects` :) might be worth throwing `helpers` on there?